cg/meta/workflow/taxprofiler.py

```diff
@@ -110,18 +127,73 @@ def write_sample_sheet(
                 ),
             )
 
+    def write_params_file(self, case_id: str, dry_run: bool = False) -> None:
+        """Write params-file for taxprofiler analysis in case folder."""
+        default_options: Dict[str, str] = self.get_default_parameters(case_id=case_id)
+        if dry_run:
+            return
+        NextflowAnalysisAPI.write_nextflow_yaml(
+            content=default_options,
+            file_path=NextflowAnalysisAPI.get_params_file_path(
+                case_id=case_id, root_dir=self.root_dir
+            ),
+        )
+
+    def get_reference_path(self) -> Path:
```

Comment:
unless you want to add the possibility to modify the reference path you don't need this. You can add this directly when initialiting the attributes
```
self.hostremoval_reference: Path = Path(config.taxprofiler.hostremoval_reference)
```

Reply:
Done